README.md

```diff
@@ -174,7 +174,7 @@ To use the GitHub Action with Codacy integration:
 The Codacy GitHub Action is a wrapper for running the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli) and supports [the same parameters as the command `analyze`](https://github.com/codacy/codacy-analysis-cli#commands-and-configuration), with the following exceptions:
 
 -   `--commit-uuid` (the action always analyzes the commit that triggered it)
--   `--api-token`, `--username`, and `--project` (use [`--project-token`](https://github.com/codacy/codacy-analysis-cli#project-token) instead)
+-   `--username` and `--project` (the action automatically uses the owner and name of the repository when you specify the parameter `api-token`)
```

Comment:
@rtfpessoa looking at https://github.com/codacy/codacy-analysis-cli-action/blob/master/action.yml#L89 and https://github.com/codacy/codacy-analysis-cli-action/blob/master/action.yml#L239 I understood that the GitHub Action uses the owner of the repository (or organization name) as the `--username`. But on Codacy, the account API tokens are defined at the user level. What if these names don't match? Should we be more specific in the instructions above and require the owner to create the account API token? :thinking:

Reply:
the `--username` flag represents the organization name on Codacy, which with sync organizations has the same name as the organization on GitHub - that's with the actions are assuming the value

any user that has access to the organization on Codacy will be able to interact with it using an account API token

Reply:
This is quite interesting. Thanks for the context! :+1: